addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -301,15 +308,6 @@ export class Powerbox {
         if (this._currentOpenOptions.forceShow) showOnceOnKeyup();
     }
 
```

Comment:
I'm guessing this change is unrelated and the function was just unused. I wouldn't remove it though because it's a public API function and there might be customers who use it in their custom code.